doc/Programming-guide/Using-Kaa-endpoint-SDKs/C++/SDK-Edison/index.md

```diff
@@ -81,6 +81,21 @@ Refer to [the official Edison getting started guide](https://software.intel.com/
    opkg install sqlite3
    ```
 
+7. Create .profile file and set LD_LIBRARY_PATH.
+
+   Add a profile file.
+   ```
+   nano .profile
+   ```
+   Write the following line into the added profile file and then save the file.
+   ```
+   export LD_LIBRARY_PATH=/usr/local/lib
+   ```
```

Comment:
@rasendubi , I meant the "compilation phase". We run avrocpp during the building of the demo and according to logs in http://jira.kaaproject.org/browse/APP-271, it cannot find run-time libraries.
Yes, of course, we also need run-time libraries to run the application.